data/yam/autoyast/support_images/create_hdd_sles15sp6_ha_aarch64.xml.ep

```diff
@@ -0,0 +1,329 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!DOCTYPE profile>
+<profile xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
+  <bootloader config:type="map">
+    <global config:type="map">
+      <cpu_mitigations>auto</cpu_mitigations>
+      <gfxmode>auto</gfxmode>
+      <hiddenmenu>false</hiddenmenu>
+      <os_prober>false</os_prober>
+      <secure_boot>true</secure_boot>
+      <terminal>gfxterm</terminal>
+      <timeout config:type="integer">-1</timeout>
+      <update_nvram>true</update_nvram>
+      <xen_kernel_append>vga=gfx-1024x768x16</xen_kernel_append>
```

Comment:
As my understanding, only reserve secure_boot and timeout is enough.

Reply:
I don't get what is the suggested solution. I diff the global section with another XML files and all are pretty the same

create_hdd_hpc_textmode_aarch64.xml.ep
```
    <global t="map">
      <append>splash=silent plymouth.ignore-serial-consoles console=ttyAMA0 console=tty kernel.softlockup_panic=1 quiet security=apparmor mitigations=auto</append>
      <cpu_mitigations>auto</cpu_mitigations>
      <gfxmode>auto</gfxmode>
      <hiddenmenu>false</hiddenmenu>
      <os_prober>false</os_prober>
      <secure_boot>true</secure_boot>
      <terminal>gfxterm</terminal>
      <timeout t="integer">-1</timeout>
      <update_nvram>true</update_nvram>
      <xen_kernel_append>vga=gfx-1024x768x16</xen_kernel_append>
    </global>
```

create_hdd_sles15sp6_ha_aarch64.xml.ep
```
    <global config:type="map">
      <cpu_mitigations>auto</cpu_mitigations>
      <gfxmode>auto</gfxmode>
      <hiddenmenu>false</hiddenmenu>
      <os_prober>false</os_prober>
      <secure_boot>true</secure_boot>
      <terminal>gfxterm</terminal>
      <timeout config:type="integer">-1</timeout>
      <update_nvram>true</update_nvram>
      <xen_kernel_append>vga=gfx-1024x768x16</xen_kernel_append>
    </global>
```

Reply:
I mean some setting may not needed for our test, while leave it there may not block the test also, do this just to compact the profile. Anyway, depends on you, both ok for me.